lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
I am a bit confused by this comment.

> 1- history is only a wrapper for the rest channel history method. It therefore takes the same options as the REST history call. 

If you look at the [slightly out of date docs](http://www.rubydoc.info/gems/ably/0.7.5/Ably/Realtime/Channel#history-instance_method) it does provide documentation for all the options.

>  In addition since it is evented, you can pass a block as a callback once the REST call is complete.

Every method in the Realtime library is a deferrable and yields a block as well, this is again quite clear in http://www.rubydoc.info/gems/ably/0.7.5/Ably/Realtime/Channel#history-instance_method

>  You can use the :until_attach option if you need to display history with a guarantee that no message will be missed.

The comments say "Once attached to a channel, you can retrieve messages published on the channel before the channel was attached with the option <tt>until_attach: true</tt>.  This is very useful for developers who wish to subscribe to new realtime messages yet also display historical messages with the guarantee that no messages have been missed."

> To use until_attach you need to make sure that the channel is attached before calling history.

It says that in the option description: ":until_attach  When true, request for history will be limited only to messages published before this channel was attached. Channel must be attached."

> 3- If the channel is not attached, this option will throw an exception.

As above